### PR TITLE
[radicle-community-tokens] add strategy: radicle-funding-nft, community tokens

### PR DESCRIPTION
**Pull request by AmirSarraf**
Changes proposed in this pull request:
- adding radicle-funding-nft strategy to list of strategies and  strategies/index.ts, using funding-subgraph-v4 of radicle.

### CODE REVIEW

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Hello @AmirSarraf 

Revert changes from package-lock.json

File: src/strategies/radicle-funding-nft/index.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import { BigNumber } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { getAddress } from '@ethersproject/address';
+import { subgraphRequest } from '../../utils';
+
+const FUNDING_SUBGRAPH_URL = {
+  '1': 'https://api.studio.thegraph.com/query/9578/funding-subgraph-v4/v0.0.2'
```
Comment (ChaituVR) [inline src/strategies/radicle-funding-nft/index.ts:7]: Is this graph a standard for radicle funding NFT? maybe we should have a way to pass this from params (so users can their own subgraph)

File: src/strategies/radicle-funding-nft/index.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+};
+
+export async function strategy(
```
Comment (ChaituVR) [inline src/strategies/radicle-funding-nft/index.ts:10]: index.ts should also export author and version
```
export const author = 'AmirSarraf';
export const version = '0.1.0';
```

example:
https://github.com/snapshot-labs/snapshot-strategies/blob/master/src/strategies/erc20-balance-of/index.ts

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/radicle-funding-nft/index.ts (hunk 1 above)
Comment (bonustrack) [reply src/strategies/radicle-funding-nft/index.ts:7]: @ChaituVR It's fine, this strategy and subgraph are specific to Radicle, the url can be hardcoded.

**Comment by bonustrack** [comment]
@AmirSarraf Hi, thanks for your contribution, how did you find this, are you aware there is a bounty on Gitcoin for this strategy?

File: src/strategies/radicle-funding-nft/index.ts (hunk 1 above)
Comment (AmirSarraf) [reply src/strategies/radicle-funding-nft/index.ts:7]: I asked them if it's a standard subgraph, no response yet

**Comment by AmirSarraf** [comment]
yes, actually i found this bounty from Gitcoin,
I added a link to the PR at Gitcoin

File: src/strategies/radicle-funding-nft/index.ts (hunk 2 above)
Comment (AmirSarraf) [reply src/strategies/radicle-funding-nft/index.ts:13]: done

**Comment by bonustrack** [comment]
@AmirSarraf Looks good! Can you just rename the strategy to "radicle-community-tokens" ? Thanks!

File: src/strategies/radicle-funding-nft/examples.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+      "name": "radicle-funding-nft",
+      "params": {
+        "address": "0x6b175474e89094c44da98b954eedeac495271d0f",
+        "symbol": "DAI",
+        "decimals": 18
+      }
+    },
+    "network": "1",
+    "addresses": [
+      "0xb5bb9a125c2f67f1f2cd9d8992955bb209490afe"
```
Comment (bonustrack) [inline src/strategies/radicle-funding-nft/examples.json:14]: Can you checksum this address also?
```suggestion
      "0xb5bb9A125c2F67F1F2cd9d8992955bb209490aFE"
```

File: src/strategies/radicle-funding-nft/examples.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "name": "Example query",
+    "strategy": {
+      "name": "radicle-funding-nft",
+      "params": {
+        "address": "0x6b175474e89094c44da98b954eedeac495271d0f",
```
Comment (bonustrack) [inline src/strategies/radicle-funding-nft/examples.json:7]: Can you make this work with checksummed address too?
```suggestion
        "address": "0x6B175474E89094C44Da98b954EedeAC495271d0F",
```

Comment (bonustrack) [inline src/strategies/radicle-funding-nft/examples.json:7]: These params aren't used, there should be a way to define the project NFT address, currently the strategy return funding amount for all projects, the strategy should give voting power for a specific project / NFT.

Comment (AmirSarraf) [reply src/strategies/radicle-funding-nft/examples.json:7]: do we need to query also for a specific NFT, or only adding the capability to query for a `fundingProject` is required?

Comment (bonustrack) [reply src/strategies/radicle-funding-nft/examples.json:7]: Only `fundingProject`

Comment (AmirSarraf) [reply src/strategies/radicle-funding-nft/examples.json]: I was receiving an empty `fundingProject` field for `nfts` query.
so for finding `nfts` of a certain `fundingProject` i added another query parameters to query `fundingProjects` (but only one subgraph query is ever requested).

**Review by bonustrack — APPROVED** [review]

**Review by ChaituVR — APPROVED** [review]

**Comment by ChaituVR** [comment]
Strategy is live: https://snapshot.org/#/strategy/radicle-community-tokens

**Comment by bonustrack** [comment]
@AmirSarraf There seem to be some issue with the strategy when we test it on the playground here: https://snapshot.org/#/playground/radicle-community-tokens
The strategy fail to resolve some time or give no results. Feel free to contact us on Discord here: https://discord.snapshot.org to discuss about this issue

**Pull request merged**